l10n_eu_oss/wizard/l10n_eu_oss_wizard.py

```diff
@@ -0,0 +1,216 @@
+                ("vat_required", "=", True),
+                ("country_group_id.id", "=", eu_id.id),
+            ],
+            limit=1,
+        )
+
+    def _default_done_country_ids(self):
+        user = self.env.user
+        eu_country_group = self._get_eu_res_country_group()
+        return (
+            eu_country_group.country_ids
+            - self._default_todo_country_ids()
+            - user.company_id.country_id
+        )
+
+    def _default_todo_country_ids(self):
+        user = self.env.user
+        eu_country_group = self._get_eu_res_country_group()
+        eu_fiscal = self.env["account.fiscal.position"].search(
+            [
+                ("country_id", "in", eu_country_group.country_ids.ids),
+                ("vat_required", "=", False),
+                ("auto_apply", "=", True),
+                ("company_id", "=", user.company_id.id),
+                ("fiscal_position_type", "=", "b2c"),
+            ]
+        )
+        return (
+            eu_country_group.country_ids
+            - eu_fiscal.mapped("country_id")
+            - user.company_id.country_id
+        )
+
+    company_id = fields.Many2one(
+        "res.company", string="Company", required=True, default=_get_default_company_id
+    )
+    done_country_ids = fields.Many2many(
+        "res.country",
+        "l10n_eu_oss_country_rel_done",
+        default=_default_done_country_ids,
+        string="Already Supported",
+    )
+    todo_country_ids = fields.Many2many(
+        "res.country",
+        "l10n_eu_oss_country_rel_todo",
+        default=_default_todo_country_ids,
+        string="EU Customers From",
+        required=True,
+    )
+    price_include_tax = fields.Boolean(string="Price Include Tax", default=False)
+    general_tax = fields.Many2one(
+        comodel_name="account.tax", string="General Tax", required=True
+    )
+    reduced_tax = fields.Many2one(comodel_name="account.tax", string="Reduced Tax",)
+    superreduced_tax = fields.Many2one(
+        comodel_name="account.tax", string="Super Reduced Tax",
+    )
+    second_superreduced_tax = fields.Many2one(
+        comodel_name="account.tax", string="Second Super Reduced Tax"
+    )
+
+    def _prepare_repartition_line_vals(self, original_rep_lines):
+        return [
+            (
+                0,
+                0,
+                {
+                    "factor_percent": line.factor_percent,
+                    "repartition_type": line.repartition_type,
+                    "account_id": line.repartition_type == "tax"
+                    and line.account_id.id
+                    or None,
+                    "company_id": line.company_id.id,
+                    "sequence": line.sequence,
+                },
+            )
+            for line in original_rep_lines
+        ]
+
+    def _prepare_tax_vals(self, country_id, tax_id, rate):
+        return {
+            "name": _("OSS for EU to {}: {}".format(country_id.name, rate)),
```

Comment:
Incorrect parenthesis for the translation and you shouldn't use format, but old syntax.